src/cascadia/TerminalApp/TerminalSettings.h

```diff
@@ -69,6 +69,7 @@ namespace winrt::TerminalApp::implementation
         GETSET_PROPERTY(bool, CopyOnSelect, false);
 
         GETSET_PROPERTY(Windows::Foundation::IReference<uint32_t>, TabColor, nullptr);
+        GETSET_PROPERTY(Windows::Foundation::IReference<uint32_t>, StartingTabColor, nullptr);

```

Comment:
@DHowett or @zadjii-msft:
Should we have a comment here or somewhere describing the difference between these two properties? I understand that we _need_ `StartingTabColor`, but I have a feeling it'll be confusing to have both down the road.

Reply:
@carlos-zamora - I totally agree with you that it is both confusing and error-prone. Instead I would like to:
1. Introduce "ITerminalCreationParams" and move this property there.
2. Pass them upon construction

However this will require changing the Terminal interface, that I felt somewhat above my paygrade at this  stage.